ui/statistics/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
     implementation project(':event')
     implementation project(":model")
     implementation project(":ui:common")
+    implementation project(path: ':ui:app-start-intent')
```

Comment:
Please use the same code style as the line above and keep the lines ordered alphabetically.